xl/trax/track.py

```diff
@@ -222,7 +222,7 @@ def __init__(self, uri=None, scan=True, _unpickles=None):
                 state. not for normal use.
         """
         # don't re-init if its a reused track. see __new__
-        if self._init == False:
+        if not self._init:
```

Comment:
Yes, that's true. Your solution has better readability than the pep8-suggested default. Will fix it.